src/content/en/tools/chrome-devtools/iterate/inspect-styles/animations.markdown

```diff
@@ -0,0 +1,160 @@
+
+The Animation Inspector opens up as a tab next to the Console Drawer. Since
+it's a Drawer tab, you can use it from any DevTools panel. 
+
+![Empty Animation Inspector](imgs/empty-ai.png)
+
+The Animation Inspector is grouped into four main sections (or panes). This
+guide refers to each pane as follows:
+
+1. **Controls**. From here you can clear all currently captured Animation 
+   Groups, or change the speed of the currently selected Animation Group. 
+2. **Overview**. Select an Animation Group here to inspect 
+   and modify it in the **Details** pane.
+3. **Timeline**. Pause and start an animation from here, or jump to a specific 
+   point in the animation.
+4. **Details**. Inspect and modify the currently selected
+   Animation Group. 
+
+![annotation Animation Inspector](imgs/annotated-animation-inspector.png)
+
+To capture an animation, just perform the interaction that triggers the
+animation while the Animation Inspector is open. If an animation is triggered 
+on page load, you can help the Animation Inspector detect the animation by 
+reloading the page. 
+
+{% animation /web/tools/chrome-devtools/iterate/inspect-styles/animations/capture-animations.mp4 %}
+
+## Inspect animations {#inspect}
+
+Once you've captured an animation, there are a few ways to replay it:
+
+* Hover over its thumbnail in the **Overview** pane to view a preview of it.
+* Select the Animation Group from the **Overview** pane (so that it's displayed
+  in the **Details** pane) and press the **replay** button
+  (![replay button](imgs/replay-button.png){:.inline}). The animation is
+  replayed in the viewport. 
+  Click on the **animation speed** buttons (![animation speed 
+  buttons](imgs/animation-speed-buttons.png){:.inline}) to change the preview 
+  speed of the currently selected Animation Group. You can use the red 
+  veritcal bar to change your current position. 
+* Click and drag the red vertical bar to scrub the viewport animation. 
+
+### View animation details
+
+Once you've captured an Animation Group, click on it from the **Overview** pane 
+to view its details. In the **Details** pane each individual animation gets
+its own row. 
+
+![animation group details](imgs/animation-group-details.png)
+
+Hover over an animation to highlight it in the viewport. Click on the animation
+to select it in the **Elements** panel. 
+
+![hover over animation to highlight it in 
+viewport](imgs/highlight-animation.png)
+
+For some animations, you may notice that the left section of the 
+visualization is colored darker than the right section. In other words, the
+right sections are more transparent than the left section. This is because the
+left section represents the animation definition and the right sections 
+represent iterations. Any edit that you make to the left section section is 
+immediately reflected in all the subsequent iterations. For example, in the 
+screenshot below, sections two and three represent iterations of section one. 
```

Comment:
This paragraph is a bit wordy and could be a lot more concise. Maybe something like "Subsequent iterations are represented as faded". The end points of each iteration are also denoted by the dark, filled circles.